src/site/content/en/blog/value-of-speed/index.md

```diff
@@ -0,0 +1,205 @@
+tags:
+  - post
+  - e-commerce
+  - analytics
+---
+
+After a lot of hard work, you've done it. You've made your company's site
+noticeably faster. Now it's time for the fun part: showing stakeholders how much
+extra revenue your work has generated!
+
+In this post we'll walk through how to do that by calculating the _relative
+mobile conversion rate_. This metric is useful because it quantifies the
+effects of site improvements while excluding external factors like marketing
+campaigns, which can obscure your findings. Let's get started!
+
+## Relative Mobile Conversion Rate (Rel mCvR)
+
+A site's conversion rate can be influenced by site characteristics—like speed
+and usability—and by external factors—like marketing campaigns, seasonal events,
+and the mix of marketing channels.
+
+{% Aside %}
+A _conversion_ happens when a site visitor becomes a customer. What counts
+as a conversion will vary depending on the nature of the site. Buying a
+product, subscribing to a service, registering as a user, or even just
+viewing a particular page could be considered a conversion. Check with your
+marketing team if you're not sure how conversions are counted in your company.
+{% endAside %}
+
+Since you're interested in how site speed affects conversions, the mobile site
+is most relevant—that's where you're most likely to see the benefits of
+speed improvements. Rather than looking at the mobile conversion rate alone,
+though, you'll be analyzing the _relative_ mobile conversion rate (Rel mCvr),
+which is calculated by dividing the mobile conversion rate by the desktop
+conversion rate. This approach reduces the noise from external factors, which
+tend to affect both desktop and mobile, and makes it easier to see whether any
+increases in the mobile site's effectiveness were actually caused by the speed
+improvements.
+
+<img class="w-screenshot" src="./relative-versus-absolute.jpg" alt="Table showing comparison of mobile/desktop conversion rate and relative mobile conversion rate">
+
+{% Aside 'caution' %}
+Rel mCvR is influenced not only by speed, but also by other site
+characteristics like usability. If other big changes were made to the site
+during the period you want to analyze, you won’t be able to measure the effects
+of speed improvements on their own, but you can show the benefits of all the
+improvements as a group.
+{% endAside %}
+
+## Doing the analysis
+
+If you have access to your site's page analytics data, you can follow along
+using that. In this walkthrough you'll be using Google Analytics to get site
+data. If you don't have a Google Analytics account, you can learn how to set one
+up at [Get started with Analytics](https://support.google.com/analytics/answer/1008015?hl=en).
+Or, if you just want to see how Rel mCvR is calculated, you can get the sample
+data set shown below from
+[this Google Sheet](https://docs.google.com/spreadsheets/d/1Mxmy1luPBOvJneSM9g2NZt6yHKx8xkpBXygOpm4LgNM/edit?usp=sharing)
```

Comment:
```suggestion
[this Google Sheet](https://docs.google.com/spreadsheets/d/1qDtWJHJptk8pwXT4f6sNyUUeyep_com2E9HIQbFxAe4/edit?usp=sharing)
```

Changed to our public Drive

Reply:
I took a quick look at the changes--I think they're good. I also tweaked a couple uses of 2nd person that felt a little clunky.